### PR TITLE
refactor: remove `Swiper` & cleanup `ToastContainer`

### LINKED ISSUE #7219: [Task]: Cleanup `ToastContainer` & remove `Swiper`

### PR DESCRIPTION
**Pull request by VmMad**
## Summary

Fixes #7219 
> - Rename component to NotificationManager
> - Cleanup classes, styles. Use grid instead of flex to be able to click between the notifications
...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by cpl121 — CHANGES_REQUESTED** [review]
I think the `Swiper` component is no longer used, and it could be removed ✂️  wdyt?

**Review by cpl121 — APPROVED** [review]
NJ!

File: packages/desktop/App.svelte [hunk 1]
```diff
@@ -199,7 +199,7 @@
             {#if settings}
                 <Settings handleClose={() => (settings = false)} />
```
Comment (evavirseda) [inline packages/desktop/App.svelte:200]: it would be great if you could fix this warning :pray:

File: packages/desktop/App.svelte [hunk 2]
```diff
@@ -199,7 +199,7 @@
...
             {/if}
-            <ToastContainer classes="absolute right-5 bottom-5 w-100" />
+            <NotificationManager />
```
Comment (evavirseda) [inline packages/desktop/App.svelte:202]: nice name :wink:

**Review by evavirseda — CHANGES_REQUESTED** [review]
nice jean, just one suggestion!!

File: packages/desktop/App.svelte (hunk 2 above)
Comment (VmMad) [reply packages/desktop/App.svelte:202]: :wink:

**Review by evavirseda — APPROVED** [review]
LGTM :rocket:

File: packages/desktop/App.svelte (hunk 2 above)
Comment (begonaalvarezd) [reply packages/desktop/App.svelte:201]: we lost a precious opportunity here to have called this component `Toaster` :trollface:

**Pull request merged**